### DISCUSSION TITLE
在启用RIFE_STD时控制台有些提示不知道是什么意思

### DISCUSSION THREAD

**Discussion by redomCL** [hooke007/mpv_PlayKit#391] [mpv 通用]
在启用RIFE_STD时控制台提示：
[vapoursynth] Frame requested during init! This is unsupported.
[vapoursynth] Returning black dummy frame with 0 duration.
补帧效果正常，画面也没有错误，这两个提示是什么意思？某些错误吗？
怕是自己配置有问题，然后用mpv-lazy-20231101.exe直接播放视频发现控制台里依然有这个提示

**Comment by redomCL** [comment]
![QQ截图20231124174003](https://github.com/hooke007/MPV_lazy/assets/22269254/2fc28362-f25f-42d5-9321-7f77c31834c3)

**Comment by hooke007** [comment]
正常，无视

**Reply by redomCL** [reply]
想请教一下这个是什么意思，大佬方便讲解一下不或者给个关键词我去学习下

**Reply by hooke007** [reply]
你可以直接搜mpv的issue
就是实现问题，翻译一下英文字面意思就是它表达的意思
